### ISSUE TITLE
Buscar dados via SNMP

### ISSUE THREAD

**Issue by naynivek** [anderson-raber/olt_zte#1]
Olá !

Já pensou em buscar os dados via SNMP ? É mais simples do que com script.

**Comment by anderson-raber** [comment]
sim, ja fiz outros dois via snmp https://share.zabbix.com/cat-app/lld-olt-zte
https://share.zabbix.com/cat-app/olt-zte-c320

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
